src/Types.ts

```diff
@@ -305,7 +317,7 @@ export interface Eta {
 export type AsyncTaskStatus = 'done' | 'pending'
 
 export interface AsyncTaskData {
-    args: any
+    args: never
```

Comment:
could you please explain why we're using never here, for both of these types? Thanks!

Reply:
alright, just wanted to make sure we actually never use/read that value. Cause I was not aware